article/_posts/2023-02-09-fully-automated-QC-for-cardiac-CMR.md

```diff
@@ -0,0 +1,101 @@
+---
+layout: review
+title: "Fully Automated, Quality-Controlled Cardiac Analysis From CMR"
+tags: cardiac 
+author: "Arnaud Judge"
+cite:
+    authors: "Bram Ruijsink, Esther Puyol-Antón, Ilkay Oksuz, Matthew Sinclair, Wenjia Bai, Julia A. Schnabel, Reza Razavi, Andrew P. King"
+    title:   "Fully Automated, Quality-Controlled Cardiac Analysis From CMR"
+    venue:   "JACC: Cardiovascular Imaging"
+pdf: "https://www.jacc.org/doi/pdf/10.1016/j.jcmg.2019.05.030"
+---
+
+
+# Introduction
+
+Clinical use of deep learning algorithms in cardiac image segmentation analysis is limited due to unavoidable issues with image quality, artifacts and anatomical variation.
+
+The objective was to develop robust quality control measures in order to facilitate translation of deep learning methods of CMR image analysis into clinical pratice.
+
+# Methods
+
+## Pipeline
+
+![](/article/images/cardiacQCpipeline/cardiacCMR_QC_pipeline.jpg)
+
+## Step 1: Pre-analysis image QC (QC1)
+
+The first quality control section aims to detect and reject bad quality images based on two categories: motion artifacts and erroneous planning.
+
+In both cases ROI extraction is used to deal with variable image size, in order to avoid issues with mofication of image quality in the processing steps. Input size was 80x80 pixels and 64x64 pixels for motion artifact detection and erroneous planning respectively.
+
+Motion artifact detection
+- 2D CNN with LSTM layer
+- k-space corruption for data augmentation to provide balanced classes. In the fourrier space, change some lines from one image slice with lines from other slices.
+![](/article/images/cardiacQCpipeline/k-space-transformation.jpg)
+<!-- - https://arxiv.org/pdf/1808.05130.pdf -->
+
+Detection of erroneous planning of 4 chamber view
+- Classification of presence of left ventricular outflow tract (LVOT) in images (246 CMR images from UK Biobank).
+- 5 layer, 2D CNN (LeNet architecture) outperformed other classifiers such as K-Nearest Neigbours, SVM, Random Forest, etc.
+<!-- - https://ieeexplore.ieee.org/stamp/stamp.jsp?tp=&arnumber=8363616 -->
```

Comment:
Maybe this could be put as a reference, otherwise it will not be visible as it is an HTML comment. See an example here:
https://github.com/vitalab/vitalab.github.io/blob/master/how-to-contribute.md#add-your-own-posts

Reply:
Done in 51d9c8d.